### PR TITLE
Cleanup after api increase

### PR DESCRIPTION
**Pull request by vbauzysvmware**
This PR cleanup things which was for 9.0(API 29.0), 9.1(API 30.0) vCD versions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-## 2.7.0 (Unreleased)
+## 2.8.0 (Unreleased)
+
+* Removed code which handled specific cases for API 29.0 and 30.0. Code now works with vCD versions: 9.5-10.1
```
Comment (lvirbalas) [inline CHANGELOG.md:3]:
```suggestion
* Removed code which handled specific cases for API 29.0 and 30.0. Supported VCD versions now: 9.5 - 10.1 
```

Comment (vbauzysvmware) [reply CHANGELOG.md]: updated

**Comment by vbauzysvmware** [comment]
> LGTM, please run the whole test suite on 9.5 before merging.

validated with 9.1 and 10.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-## 2.7.0 (Unreleased)
+## 2.8.0 (Unreleased)
+
+* Removed code which handled specific cases for API 29.0 and 30.0. Supported VCD versions now: 9.5 - 10.1
```
[the remark this answers is not in the document]
Comment (vbauzysvmware) [reply CHANGELOG.md]: updated

**Pull request merged**